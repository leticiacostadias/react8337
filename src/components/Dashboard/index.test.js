import React from 'react';
import Dashboard from './index';
import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';

describe('Dashboard', () => {
  it('deve renderizar filhos', () => {
    const dashboard = renderer.create(
      <Dashboard>
        <h1>Título do dashboard</h1>
      </Dashboard>
    );
    const arvoreDOM = dashboard.toJSON();
  
    expect(arvoreDOM).toMatchSnapshot();
  });

  it('deve adicionar classe de posicionamento', () => {
    const dashboard = shallow(
      <Dashboard posicao="centro">
        <h1>Título do dashboard</h1>
      </Dashboard>
    );
    const dashboardCentro = dashboard.find('.dashboard__centro');

    expect(dashboardCentro.length).toBe(1);
  });
});