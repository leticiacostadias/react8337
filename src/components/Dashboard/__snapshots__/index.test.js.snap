// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dashboard deve renderizar filhos 1`] = `
<div
  className="dashboard dashboard__undefined"
>
  <h1>
    TÃ­tulo do dashboard
  </h1>
</div>
`;
